lib/timers.js

```diff
@@ -439,7 +439,7 @@ exports.clearInterval = function(timer) {
 };
 
 
-const Timeout = function(after) {
+function Timeout(after) {
```

Comment:
Well, it modernizes things a little and makes them more explicit. If we want to minimize change why not keep it `const Timeout` and do `const Timeout = function Timeout` - by naming the function expression you get the same trace.